plugins.rst

```diff
@@ -22,11 +22,15 @@ Overview
 
 The following is the list of official plugins by |omv|.
 
-* **ClamAV**: Provides antivirus scan for folders.
+* **ClamAV**: Provides Clam AntiVirus (ClamAV). It is a free software, cross-platform and open-source antivirus software toolkit able to detect many types of malicious software, including viruses
+* **Diskstats**: Complementary plugin to extend system statistics collection by adding iostats graphs.
 * **Forked-daap**: Provides a daap protocol music server.
 * **LVM2**: LVM managing. Create volume groups and logical partitions.
-* **Diskstats**: Complementary plugin to extend current system statistics collection by adding iostats graphs.
 * **NUT**: Controlling and configuring UPS. The driver support is based on NUT.
+* **ShAirPort**: Provides Airtunes emulator. Stream music wirelessly to your iPod/iPad/iPhone/iTunes.
```

Comment:
I'm not sure, but i think it is named `Shairport`, at least according to the [Git repository](https://github.com/mikebrady/shairport-sync) of the project.